cd-db/Dockerfile

```diff
@@ -1,4 +1,7 @@
-FROM postgres:alpine
-MAINTAINER butlerx <butlerx@notthe.cloud>
-
-ADD setup_zen.sh   /docker-entrypoint-initdb.d/10-setup_zen.sh
+FROM postgres:9-alpine
+LABEL maintainer="butlerx <cian@coderdojo.org>"
+COPY dumps /db
```

Comment:
Why using a copy over a volume? 
Copy means you'll cp a 2.5G file in your image, while a volume is a symlink-link ?

Reply:
had issues with permissions of the dumps that why the next line chownes them